python/hail/expr/functions.py

```diff
@@ -2762,6 +2737,32 @@ def struct(**kwargs):
     """
     return to_expr(Struct(**kwargs))
 
+def tuple(args):
```

Comment:
Should this be varargs?

Reply:
I'm with @tpoterba:

```
>>> tuple([1, 2, 3])
(1, 2, 3)
>>> tuple(1, 2, 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: tuple() takes at most 1 argument (3 given)
```

Reply:
Ok. I think the docs need to be clearer for what parameters are accepted. The builtin accepts any iterable object in Python. Looks like lists, sets, tuples all behave as expected. Dictionaries return a tuple of the keys.

Reply:
yeah, the docs are actually just wrong as written.